### PR TITLE
feat(ipfs-topology): map elp network to mainnet peerlists

### PR DESCRIPTION
**Pull request by valmack**
Support for `elp` network. Using switch statements to avoid code duplication (aka human error)

### CODE REVIEW

File: packages/ipfs-topology/src/ipfs-topology.ts [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+const PEER_FILE_URLS = (ceramicNetwork: string): string | null => {
+  switch (ceramicNetwork) {
+    case Networks.MAINNET:
+    case Networks.ELP:
+      return "https://raw.githubusercontent.com/ceramicnetwork/peerlist/main/mainnet.json";
+    case Networks.TESTNET_CLAY:
+      return "https://raw.githubusercontent.com/ceramicnetwork/peerlist/main/testnet-clay.json";
+    case Networks.DEV_UNSTABLE:
+      return "https://raw.githubusercontent.com/ceramicnetwork/peerlist/main/dev-unstable.json";
+    default:
```
Comment (stbrody) [inline packages/ipfs-topology/src/ipfs-topology.ts:14]: If there's a `default` case then you don't get compile time checking that no case was missed, at which point using a switch statement doesn't seem to add value over the object form that was there previously.  Probably should drop the default case and add a case for in-memory and local networks that just explicitly returns null

Comment (valmack) [reply packages/ipfs-topology/src/ipfs-topology.ts:17]: Updated this to add a default unhandled case.

Missing a case

> src/ipfs-topology.ts:18:13 - error TS2322: Type 'import("/Users/val/Code/js-ceramic/packages/common/lib/networks").Networks' is not assignable to type 'never'.
> 18       const unhandledCase: never = ceramicNetwork;
               ~~~~~~~~~~~~~
> Found 1 error.

Not missing a case, it builds successfully.

Comment (stbrody) [reply packages/ipfs-topology/src/ipfs-topology.ts]: weird.  Oh javascript, always so wonky. Glad you were able to figure out how to make it work though!

**Review by stbrody — APPROVED** [review]
LGTM

**Pull request merged**